pkg/cmd/tasks/deploy/script.go

```diff
@@ -0,0 +1,131 @@
+package deploy
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/airplanedev/cli/pkg/api"
+	"github.com/airplanedev/cli/pkg/build"
+	"github.com/airplanedev/cli/pkg/logger"
+	"github.com/airplanedev/cli/pkg/runtime"
+	"github.com/airplanedev/cli/pkg/taskdir/definitions"
+)
+
+// DeployFromScript deploys from the given script.
+func deployFromScript(ctx context.Context, cfg config) error {
+	var client = cfg.client
+	var ext = filepath.Ext(cfg.file)
+
+	if ext == "" {
+		return errors.New("cannot deploy a file without extension")
+	}
+
+	r, ok := runtime.Lookup(cfg.file)
+	if !ok {
+		return fmt.Errorf("cannot deploy a file with extension of %q", ext)
+	}
+
+	code, err := ioutil.ReadFile(cfg.file)
+	if err != nil {
+		return fmt.Errorf("reading %s - %w", cfg.file, err)
+	}
+
+	slug, ok := r.Slug(code)
+	if !ok {
+		return errors.New("")
+	}
+
+	task, err := client.GetTask(ctx, slug)
+	if err != nil {
+		return err
+	}
+
+	def, err := definitions.NewDefinitionFromTask(task)
+	if err != nil {
+		return err
+	}
+
+	buildkind, err := build.ToBuilderKind("local")
+	if err != nil {
+		return err
+	}
+
+	abs, err := filepath.Abs(cfg.file)
+	if err != nil {
+		return err
+	}
+
+	def.Node = &definitions.NodeDefinition{}
+
+	resp, err := build.Run(ctx, build.Request{
+		Builder: buildkind,
+		Client:  client,
+		TaskID:  task.ID,
+		Root:    filepath.Dir(abs),
+		Def:     def,
+		TaskEnv: def.Env,
+	})
+	if err != nil {
+		return err
+	}
+
+	kind, kindOptions, err := def.GetKindAndOptions()
+	if err != nil {
+		return err
+	}
+
+	_, err = client.UpdateTask(ctx, api.UpdateTaskRequest{
+		Slug:             def.Slug,
+		Name:             def.Name,
+		Description:      def.Description,
+		Image:            resp.ImageURL,
+		Command:          []string{"node", cfg.file},
```

Comment:
Let's leave this as-is -- it'll be `[]` which instructs the agent to default to the the entrypoint from inside the Docker image.

```suggestion
		Command:          def.Command
```

Reply:
There's no `def.Command`, changed to `[]`.